Frontend/src/pages/dashboard/components/Customer/RecentOrder.jsx

```
@@ -117,12 +119,23 @@ const RecentOrders = ({ fullList = false, customerId }) => {
-                  {order.items.map((item) => `${item.name} (x${item.quantity})`).join(", ")}
+                  {order.items?.map((item) => `${item.name} (x${item.quantity})`).join(", ") || "No items"}
                 </TableCell>
-                <TableCell>${order.total.toFixed(2)}</TableCell>
-                <TableCell>{order.payment}</TableCell>
                 <TableCell>
-                  <OrderStatus status={order.status} />
+  {(() => {
+    const cleanTotal = typeof order.total === "string"
+      ? order.total.replace(/[^0-9.]/g, "")
+      : order.total;
+    return isNaN(parseFloat(cleanTotal))
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)